uds/client.py

```
@@ -112,6 +90,28 @@ def __init__(self,
         self.__last_functional_request: None | UdsMessageRecord = None
         self.__last_functional_response: None | UdsMessageRecord = None
         self.__last_tester_present_requests: list[UdsMessageRecord] = []
+        # TIMING PARAMETERS
+        self.__p2_client_measured: None | TimeMillisecondsAlias = None
```

Ruff (0.16.1)

[warning] `None` not at the end of the type union.

Move `None` to the end of the type union

(RUF036)